src/libs/GroupChatUtils.ts

```diff
@@ -7,7 +7,7 @@ import * as ReportUtils from './ReportUtils';
  * Returns the report name if the report is a group chat
  */
 function getGroupChatName(report: OnyxEntry<Report>): string | undefined {
-    const participants = report?.participantAccountIDs ?? [];
+    const participants = report?.participantAccountIDs?.slice(0, 5) ?? [];
```

Comment:
`getGroupChatName` function is called at multiple places, so we can't directly apply the slice here as it will affect the other places(eg. OptionRowLHN, ReportSettings). That creates a discrepancy in web and mobile LHN titles.
I think we can add an optional parameter something like `shouldApplyLimit` in this function and use it from `HeaderView`.

cc: @deetergp

Reply:
Good catch @Pujan92, that sounds like a good solution.